packages/server/postgres/migrations/2025-03-18T15:55:35.723Z_normalizeUserTms.ts

```diff
@@ -0,0 +1,26 @@
+import {sql, type Kysely} from 'kysely'
+
+// biome-ignore lint/suspicious/noExplicitAny: `any` is required here since migrations should be frozen in time. alternatively, keep a "snapshot" db interface.
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema.alterTable('User').dropColumn('tms').execute()
+}
+
+// biome-ignore lint/suspicious/noExplicitAny: `any` is required here since migrations should be frozen in time. alternatively, keep a "snapshot" db interface.
+export async function down(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .alterTable('User')
+    .addColumn('tms', sql`character varying(100)[]`, (col) => col.defaultTo('{}').notNull())
```

Comment:
`varchar(100)[]` means that the textual representation of the ENTIRE array must be < 100 chars, i.e. this will break if you join 10 or 12 teams.

Reply:
Isn't that the same what we currently have or am I missing something?
https://github.com/ParabolInc/parabol/blob/c1c69730545df887a3003de9b3dda6c4870d0cf0/packages/server/postgres/migrations/2025-01-08T00%3A27%3A52.203Z_init.ts#L2467

Reply:
🤔 THAT IS INTERESTING!
Either my understanding is wrong or we've got a problem! will investigate...